.docs/user-guide/docker-plugins.md

```diff
@@ -278,6 +278,52 @@ Environment Variable | Description | Default | Required
 `GCEPD_TAG` | Only use volumes that are tagged with a label | |
 `GCEPD_ZONE` | GCE Availability Zone | |
 
+## OpenStack
+REX-Ray ships with plug-ins for OpenStack as well.
+
+### Cinder volume
+The CINNDER plug-in can be installed with the following command:
```

Comment:
Hi @sebglon,

Please correct the spelling and casing of `CINNDER` to `Cinder`.